doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -18,26 +18,42 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
-This chapter describes in detail the Kubernetes Reference Architecture in terms of the functional capabilities and how they relate to the Reference Model requirements, i.e. how the infrastructure profiles are determined, documented and delivered.
-
-The specifications defined in this chapter will be detailed with unique identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g. `ra2.ch.001` for the first requirement in the Container Host section.  These specifications will then be used as requirements input for the Kubernetes Reference Implementation and any Vendor or Community Implementations.
-
-Figure 4-1 below shows the architectural components that are described in the subsequent sections of this chapter.
-
-<p align="center"><img src="../figures/ch04_k8s_architecture.png" alt="Kubernetes Reference Architecture" Title="Kubernetes Reference Architecture" width="65%"/></p>
+This chapter describes in detail the Kubernetes Reference Architecture in terms
+of the functional capabilities and how they relate to the Reference Model
+requirements, i.e. how the infrastructure profiles are determined, documented
+and delivered.
+
+The specifications defined in this chapter will be detailed with unique
+identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g.
+`ra2.ch.001` for the first requirement in the Container Host section.  These
+specifications will then be used as requirements input for the Kubernetes
+Reference Implementation and any Vendor or Community Implementations.
+
+Figure 4-1 below shows the architectural components that are described in the
+subsequent sections of this chapter.
+
+<p align="center"><img src="../figures/ch04_k8s_architecture.png"
+alt="Kubernetes Reference Architecture" Title="Kubernetes Reference
+Architecture" width="65%"/></p>
 <p align="center"><b>Figure 4-1:</b> Kubernetes Reference Architecture</p>
 
 <a name="4.2"></a>
 ## 4.2 Container Host
 
-This section describes the configuration that will be applied to the physical or virtual machine and an installed Operating System.
-In order for a Container Host to be conformant with the Reference Architecture it must be implemented as per the following specifications:
+This section describes the configuration that will be applied to the physical or
+virtual machine and an installed Operating System. In order for a Container Host
+to be conformant with the Reference Architecture it must be implemented as per
+the following specifications:
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.ch.001`||||
-|`ra2.ch.002`||||
-|`ra2.ch.003`||||
+|`ra2.ch.001`|Huge Pages|Huge Pages must be enabled within the Container Host OS (e.g. to verify run `cat /proc/meminfo ¦ grep Huge`)|[infra.com.cfg.004](./chapters/chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
```

Comment:
do we need to specify the huge page sizes, number, etc.?

Reply:
I did wonder about that.  There isn't anything in the RM that specifies that, only that huge pages needs to be supported.  I guess for K8s we can narrow it down to 2Mi and 1Gi, but otherwise there's nothing that tells us we need to include a quantity of each of them.  I guess for the purposes of the specification, we need to ask that at least one of each is configured and the success criteria for a test would be that at least one of each is shown as allocatable.  I'll modify as such.

Reply:
I've updated in https://github.com/cntt-n/CNTT/pull/1805/commits/2bf586b978a739fe6d39643c498d46a09c2cc6cd to read:

> Huge Pages must be enabled within the Container Host OS, with at least one each of 2048KiB and 1048576KiB page sizes, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.